eth/src/main/kotlin/com/d3/eth/deposit/main.kt

```diff
@@ -76,9 +77,11 @@ fun executeDeposit(
         depositConfig.registrationServiceIrohaAccount
     )
     val ethTokensProvider = EthTokensProviderImpl(
-        queryAPI,
-        depositConfig.tokenStorageAccount,
-        depositConfig.tokenSetterAccount
+        IrohaQueryHelperImpl(queryAPI),
```

Comment:
do we really need such a class? Query API is intended to hide Iroha API allowing to execute only query specified. We already have `RequestHelper` for similar responsibility. 🤔

Reply:
I think request helper should be replaced with IrohaQueryHelper. RequestHelper is not a class, but a set of functions, it is hard to mock.

Reply:
So it should be done atomically, otherwise it remains inconsistent :(

Reply:
Make sense.